lib/testOptions.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+
+/**
+ * Get the default mocha options for ads extension unit test. The following environment variables will be examined.
```

Comment:
`Get default mocha options for an ADS test, using the below environment variables to control what values various options are set to`

Reply:
done